.github/actions/vmtest/build.sh

```diff
@@ -2,13 +2,16 @@
 
 set -euo pipefail
 
+ARCH="$1"
```

Comment:
in run_selftests.sh we just do `ARCH=$(uname -m)`, why can't we do the same here? It seems simpler, if it works reliably.

Reply:
we in run_selftests.sh we do ARCH=$(uname -m) because there is not way to pass this information from GitHub Action. On the other hand, in the GH Action we pass this information around as a parameter. I prefer to pass it as a parameter in this case, as it will be better for cross-compilation if it ever happens.

I am not too strong about this, happy to change if you prefer